frontend/components/.eslintignore

```diff
@@ -0,0 +1,48 @@
+*.min.*
+dist
+src/directives
+src/utils
+
+!.eslintrc.js
+
+# yuche/vue-strap
+src/Accordion.vue
+src/Affix.vue
+src/Alert.vue
+src/Aside.vue
+src/buttonGroup.vue
+src/Carousel.vue
+src/Checkbox.vue
+src/Datepicker.vue
+src/Dropdown.vue
+src/FormGroup.vue
+src/index.js
+src/Input.vue
+src/main.js
+src/Modal.vue
+src/Navbar.vue
+src/Option.vue
+src/Panel.vue
+src/Popover.vue
+src/Progressbar.vue
+src/Radio.vue
+src/Select.vue
+src/Slider.vue
+src/Spinner.vue
+src/Tab.vue
+src/TabGroup.vue
+src/Tabset.vue
+src/Thumbnail.vue
```

Comment:
> Actually, I ignored .vue files in MarkBind/vue-strap#29 to avoid large code style changes

I'll open an issue for this sometime, but it's rather unrelated at the moment.
Perhaps we could use git annotate to see the % of lines changed to decide which to unignore, but as we migrate more components to bootstrap-vue this shouldn't be necessary 😄